sim/paladin/items.go

```diff
@@ -34,4 +38,76 @@ func init() {
 			character.PseudoStats.MobTypeAttackPower += 36
 		}
 	})
+
+	core.NewItemEffect(HerosBrand, func(agent core.Agent) {
+		//Increases critical strike chance of holy shock spell by 2%
+		paladin := agent.GetCharacter()
+		paladin.OnSpellRegistered(func(spell *core.Spell) {
+			if spell.SpellCode == SpellCode_PaladinHolyShock {
+				spell.BonusCritRating += 2.0
+			}
+		})
+	})
+
+	
+	core.NewItemEffect(zandalarFreethinkersBreastplate, func(agent core.Agent) {
+		character := agent.GetCharacter()		
+		character.PseudoStats.SchoolBonusHitChance[stats.SchoolIndexHoly] += 3
+	})
+
+	
+	core.NewItemEffect(zandalarFreethinkersBelt, func(agent core.Agent) {
+		character := agent.GetCharacter()		
+		character.PseudoStats.SchoolBonusHitChance[stats.SchoolIndexHoly] += 2
+	})
+
+
+	// https://www.wowhead.com/classic/item=231285/grileks-charm-of-valor
+	// Use: Increases the critical hit chance of Holy spells by 10% for 15 sec. If Shock and Awe is engraved, gain an additional 5%. (1 Min, 30 Sec Cooldown)
+	core.NewItemEffect(grileksCharmOfValor, func(agent core.Agent) {
+		character := agent.GetCharacter()
+		paladin := agent.(PaladinAgent).GetPaladin()
+
+		character.PseudoStats.BonusDamage += 4
+
+		duration := time.Second * 15
+
+		aura := character.RegisterAura(core.Aura{
+			ActionID: core.ActionID{ItemID: grileksCharmOfValor},
+			Label:    "Brilliant Light",
+			Duration: duration,
+			OnGain: func(aura *core.Aura, sim *core.Simulation) {
+				character.AddStatDynamic(sim, stats.SpellCrit, core.TernaryFloat64(paladin.hasRune(proto.PaladinRune_RuneCloakShockAndAwe), 15.0 * core.CritRatingPerCritChance, 10.0 * core.CritRatingPerCritChance))
+			},
+			OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+				character.AddStatDynamic(sim, stats.SpellCrit, core.TernaryFloat64(paladin.hasRune(proto.PaladinRune_RuneCloakShockAndAwe), -15.0 * core.CritRatingPerCritChance, -10.0 * core.CritRatingPerCritChance))
```

Comment:
It looks like the trinket only applies to holy spells. We could either create a school crit multiplier pseudostat or alternatively you could just apply the bonus crit to any of the paladin's spells matching the holy school. It's odd though the wowhead entry looks like it just gives the paladin 10% crit with no specific school

Reply:
<img width="300" alt="image" src="https://github.com/user-attachments/assets/bec7ca86-a157-4a80-a6b7-b96167e116b8">